src/extras/simplePagination.ts

```diff
@@ -0,0 +1,76 @@
+import { stringifyVariables } from 'urql';
+import { Resolver, Variables } from '../types';
+
+export interface PaginationParams {
+  fromName?: string;
+  limitName?: string;
+}
+
+interface Base {
+  id?: any;
+  _id?: any;
+  __typename: any;
+}
+
+export const simplePagination = (params?: PaginationParams): Resolver => {
+  const from = (params && params.fromName) || 'from';
+  const limit = (params && params.limitName) || 'limit';
+
+  const compareArgs = (
+    fieldArgs: Variables,
+    connectionArgs: Variables
+  ): boolean => {
+    for (const key in connectionArgs) {
+      if (key === from || key === limit) {
+        continue;
+      } else if (!(key in fieldArgs)) {
+        return false;
+      }
+
+      const argA = fieldArgs[key];
+      const argB = connectionArgs[key];
+
+      if (
+        typeof argA !== typeof argB || typeof argA !== 'object'
+          ? argA !== argB
+          : stringifyVariables(argA) !== stringifyVariables(argB)
+      ) {
+        return false;
+      }
+    }
+
+    return true;
+  };
+
+  return (_parent, fieldArgs, cache, info) => {
+    const { parentKey: key, fieldName } = info;
+    const connections = cache.resolveConnections(key, fieldName);
+    const size = connections.length;
+    const result: Base[] = [];
+    const visited = new Set();
```

Comment:
We keep a visited Set to see which nodes have been visited so we don't include duplicates when arguments overlap